src/app/create-meeting/_components/MeetingShare.tsx

```diff
@@ -1,32 +1,125 @@
 import React from 'react'
+import dayjs from 'dayjs'
+import Image from 'next/image'
+import { Button } from '@/components/Button'
+import Tooltip from '@/components/Tooltip'
+import useMeetStore from '@/stores/useMeetStore'
+import useTooltipStore from '@/stores/useTooltipStore'
 
-interface MeetingShareProps {
-  onShareMeeting: () => void
-  onGoToMyMeeting: () => void
-}
+function MeetingShare() {
+  const { activeTooltip, showTooltip, hideTooltip } = useTooltipStore()
+  const { meetingResult } = useMeetStore()
+
+  if (!meetingResult) {
+    return null
+  }
 
-function MeetingShare({ onShareMeeting, onGoToMyMeeting }: MeetingShareProps) {
   return (
     <div className="flex flex-col items-center">
-      <h2 className="text-2xl font-bold mb-4">모임이 생성되었어요</h2>
-      <p className="text-lg mb-6">미리 공유해볼까요?</p>
-      <div className="bg-gray-100 w-full aspect-square mb-6 flex items-center justify-center">
-        <p className="text-gray-400">축하 이미지</p>
+      <div className="mb-[42px] text-center">
+        <h2 className="text-xl font-bold mb-2">모임이 생성되었어요 🎉</h2>
+        <p className="text-sm text-gray-600 mb-7">
+          생성된 모임 앨범을 친구들에게 알려보세요.
+        </p>
+        <Image
+          src={meetingResult.qrCodeUrl || ''}
```

Comment:
P5

```suggestion
          src={meetingResult.qrCodeUrl || defaultImage}
```

흐름상 `defaultImage`가 필요할 것 같아요 `||`를 사용하시는 걸 싫어하신다면 `store`에서 만드는 `generateQRCode` 함수에서 대체 주소를 전달해도 될 것 같네요

```suggestion
          src={meetingResult.qrCodeUrl}
```

Reply:
defaultImage의 사용하는 것이 적합하지만 qrCode imgae가 필수값이여서!! 확인 감사합니다!